Add a GitHub Pages site

koistya opened the pull request:
1. Create an orphan [gh-pages branch](https://help.github.com/articles/creating-project-pages-manually/)
2. `npm install -g easystatic` - installs [Easystatic](https://easystatic.com) site generator, optimized for GH Pages sites
3. `npm run docs` - generates a static site from `*.md` files in the `docs` folder and launches it in a browser with "live reload" enabled (via [Browsersync](https://browsersync.io/))
4. `npm run docs:deploy` - pushes the generated static site to `gh-pages` branch

P.S.: I created [this utility](https://github.com/easystatic/easystatic) for my own OSS projects and though it might be helpful for you as well.

danyshaanan commented:
Hi @koistya.

A disclaimer would be appropriate regarding your ownership of easystatic, which seems like an odd projects in several aspects:
* Its repo seems to be created an hour before the opening of this PR.
* It has a single commit (which makes going through the code much harder).
* npm releases (seen with `npm info easystatic time`) do not correspond to tags in Github.

All of this doesn't help to create much confidence in that package or in this PR.

Thanks.

koistya commented:
@danyshaanan thanks for your comment! I have addressed some of the issues you mentioned above by creating a CHANGELOG.md file with the list of past and future/planned updates to this static site generator, improved documentation to the project, created tags matching the recently published updates on NPM.
